### PR TITLE
Fix wrong params naming in openpitrix

### PR DESCRIPTION
**Pull request by sologgfun**
### What type of PR is this?
Add one of the following kinds:
/kind bug

Optionally add one or more of the following kinds if applicable:
/kind api-change

### What this PR does / why we need it:
In the following issue, I found the applicationId in openpitrix api is misunderstanding.
https://github.com/kubesphere/kubesphere/issues/5602
In the API, the "applicationId" parameter is actually the name of the helmRelease, but it uses "applicationId" as the parameter name, as shown in the following figure.

image

### Which issue(s) this PR fixes:

Fixes # https://github.com/kubesphere/kubesphere/issues/5602

### Special notes for reviewers:

This PR will cause changes to the OpenAPI, so we need to be cautious. Additionally, the commit only made changes to the upgrade API, while in reality, the get API and others also need modification. I would like to confirm the opinions of the members. Is it necessary to make these changes?

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kapis/openpitrix/v1/handler.go
Comment (wtw1994) [inline pkg/kapis/openpitrix/v1/handler.go:1]: fix bug using   upgradeClusterRequest.AppId = applicationId  , please do not modify the naming convention for the appstore API

**Comment by sologgfun** [comment]
AppId already exist in body, and it`s different from the params in path.They have 
So `upgradeClusterRequest.AppId = applicationId` maybe is not suitable.

image

These two parameters have different meanings. Clearly, the AppId in the body refers to the AppId in the helmRelease.

image

At the same time, in order to avoid modifying the naming convention for the appstore API, I submitted a new commit.
Can you check if it complies with the design principles? @wtw1994

**Review by wtw1994 — APPROVED** [review]

**Pull request merged**